<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Saved video name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Saved csv name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_video_name">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>411</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>captured_video</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_csv_name">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>411</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>csv_record</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>1091</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Process</string>
    </property>
    <widget class="QPushButton" name="btn_process">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Ratio:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_ratio">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="video_frame">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>601</width>
       <height>411</height>
      </rect>
     </property>
     <property name="text">
      <string>l</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
